.github/workflows/tests.yml

```diff
@@ -161,14 +161,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10']
+        python-version: ['3.7', '3.10']
         conda-version: [release]
         test-type: [serial, parallel]
         include:
-          - python-version: '3.9'
+          - python-version: '3.10'
             conda-version: canary
             test-type: serial
-          - python-version: '3.9'
+          - python-version: '3.10'
```

Comment:
Maybe deal with a version we are testing under? What is this include directive for?

Reply:
`include`s are appended to the `matrix`